src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -163,6 +164,22 @@ class ReimbursementAccountPage extends React.Component {
     }
 
     render() {
+        if (this.props.network.isOffline) {
+            return (
+                <>
+                    <HeaderWithCloseButton
+                        title={this.props.translate('workspace.common.bankAccount')}
+                        onCloseButtonPress={Navigation.dismissModal}
+                        onBackButtonPress={Navigation.goBack}
+                        shouldShowGetAssistanceButton
+                        guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
+                        shouldShowBackButton
+                    />
+                    <FullPageOfflineBlockingView />
```

Comment:
We want to wrap this around the view that we want to hide. 

You can surround the code in reimbursementAccountForm

This way, buttons will stay.
But as soon as you click on it, it would show the offline view, for any of the forms